doc/README-MIC

```diff
@@ -0,0 +1,74 @@
+
+Intel MIC (Xeon Phi) is a coprocessor computer architecture developed by Intel.
+Program built for MIC cannot run on normal Intel processors. So you need to 
+have a MIC coprocessor and related enviroments setup before trying to build 
+John the Ripper (JtR) for MIC. Tutorial on how to setup the software 
+environments for MIC can be found on Intel's website.
+
+
+-----------------------
+Library Denpendencies:
+-----------------------
+
+JtR requires some libraries that are not available on MIC, which means you'll 
+need to built them for MIC by yourself.
+These libraries are:
+    Zlib (libz)
+    GMP (libgmp)
+    OpenSSL (libssl & libcrypto)
+
+They can be downloaded from their websites. But building them require some 
+effort, and only the mentioned versions are guaranteed to work.  
+Assuming those libraries are to be install under path $MIC and the path to
+JtR is $JOHN, then follow the steps below.
+
+Build Zlib (version 1.2.8):
+$ cd */zlib-1.2.8
+$ CC=icc CFLAGS="-mmic" ./configure --prefix=$MIC
```

Comment:
That's fixed now. Thank you.
BTW, I'm still working on removing the "-lssl -lcrypto" flags from LDFLAGS when building JtR for MIC.